src/WebJobs.Extensions.ApiHub/Table/ApiHubTableAttribute.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Attribute that specifies an ApiHub table or entity.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public sealed class ApiHubTableAttribute : Attribute
+    {
+        /// <summary>
+        /// Gets the key of the configuration setting that specifies the connection string.
+        /// </summary>
+        public string Key { get; }
```

Comment:
I wanted to do that but then opted for keeping it consistent with the ApiHubFileAttribute. Will change to Connection.